assets/javascripts/kitten/karl/loaders/karl-loaders.js

```diff
@@ -1,28 +1,29 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { Loader } from 'kitten/components/loaders/loader'
 import { Button } from 'kitten/components/buttons/button'
 import { Marger } from 'kitten/components/layout/marger'
 
-export const KarlLoaderInButton = props => {
-  return (
-    <div>
-      <Marger top="1" bottom="1">
-        <Button icon iconWithMinWidth>
-          <Loader className="k-Button__icon" />
-        </Button>
-      </Marger>
-
-      <Marger top="1" bottom="1">
-        <Button modifier="helium" icon iconWithMinWidth>
-          <Loader color="#fff" className="k-Button__icon" />
-        </Button>
-      </Marger>
+export class KarlLoaderInButton extends Component {
+  render() {
+    const isDisabled = this.props.disabled
+    const isModifier = this.props.modifier
 
-      <Marger top="1" bottom="1">
-        <Button icon iconWithMinWidth disabled>
+    return (
+      <Marger
+        top="1"
+        bottom="1"
+      >
+        <Button
+          icon
+          iconWithMinWidth
+          disabled={ isDisabled }
```

Comment:
On peut utiliser `{ ...this.props }` pour propager `disabled` et `modifier`. Ca évite les `const` plus haut dans la méthode.